tests/models/t5/test_modeling_flax_t5.py

```diff
@@ -1076,7 +1076,7 @@ def test_summarization(self):
         expected_summaries = [
             'prosecutor: "so far no videos were used in the crash investigation" two magazines claim to have found a'
             " cell phone video of the final seconds . \"one can hear cries of 'My God' in several languages,\" one"
-            " magazine says . all 150 on board were killed when germanwings flight 9525 crashed .",
+            " magazine says . all 150 on board were killed in the crash .",
```

Comment:
Ran slow tests on T5, GPT2, and BART, for all 3 frameworks, this was the only observed difference. It is to be noted that Flax's output for this particular test was already different from TF's and PT's outputs. Also, it fits one of the criteria for incorrect Flax results (`batch_size > 1`)